client/src/modules/stock/exit/exit.js

```diff
@@ -427,6 +429,53 @@ function StockExitController(
     };
   }
 
+  function buildDescription(entityUuid, invoiceUuid) {
+    const glb = {};
+    return PatientService.read(null, { uuid : entityUuid })
+      .then(([patient]) => {
```

Comment:
I think you can probably combine a lot of these calls together to save space and time.  For example:
```js
// we use all settled here, since any of these might be undefined.
return Promise.allSettled([
  PatientService.read(entityUuid),
  PatientInvoiceService.read(invoiceUuid),
  ServiceService.read(entityUuid),
]).then(([patient, invoice, service]) => {
  // etc..
});
```